test/Microsoft.DocAsCode.Build.Engine.Tests/DocumentBuilderTest.cs

```diff
@@ -178,45 +178,45 @@ public void TestBuild()
                     Assert.True(File.Exists(Path.Combine(_outputFolder, Path.ChangeExtension(conceptualFile, RawModelFileExtension))));
                     var model = JsonUtility.Deserialize<Dictionary<string, object>>(Path.Combine(_outputFolder, Path.ChangeExtension(conceptualFile, RawModelFileExtension)));
                     Assert.Equal(
-                        $"<h1 id=\"hello-world\" sourcefile=\"{_inputFolder}/test.md\" sourcelinenumber=\"7\">Hello World</h1>",
+                        $"<h1 id=\"hello-world\" sourcefile=\"{_inputFolder}/test.md\" sourcestartlinenumber=\"7\" sourceendlinenumber=\"7\">Hello World</h1>",
```

Comment:
it just one line , I thnk the ValidLineCount means 'validLieCount-1'

---
In reply to: [77101111](https://github.com/dotnet/docfx/pull/654#discussion_r77101111) [](ancestors = 77101111)